aiobotocore/httpsession.py

```diff
@@ -59,11 +71,13 @@ def __init__(
 
         # TODO: handle socket_options
         # keep track of sessions by proxy url (if any)
-        self._sessions: Dict[Optional[str], aiohttp.ClientSession] = {}
+        self._sessions: dict[str | None, aiohttp.ClientSession] = {}
         self._verify = verify
         self._proxy_config = ProxyConfiguration(
             proxies=proxies, proxies_settings=proxies_config
         )
+        conn_timeout: float | None
+        read_timeout: float | None
```

Comment:
nop?

Reply:
not for a type checker, otherwise the following lines will error. And you can't do
`conn_timeout: float|None, read_timeout: float|None = timeout`

the types were extremely helpful when trying to get this working and understanding the current code, but can delete these now if you don't care about type errors/want to adhere to botocore.

Reply:
deleting